radicle/src/storage/git/transport/local/url.rs

```diff
@@ -47,6 +48,16 @@ impl Url {
     pub const SCHEME: &str = "rad";
```

Comment:
Is there a reason the scheme is `"rad"` and not `"rad://"`?

Reply:
Mainly it's because we use this var to register the scheme with git2, and it expects the scheme without the `://`. I think technically speaking, `://` is also not part of the scheme, if I'm not mistaken.

Reply:
Yeah, it's just easier to add `://` to `rad` than to remove it!